temoa/types/validation_types.py

```
@@ -0,0 +1,237 @@
+        Create a failed validation result with errors.
+
+        Args:
+            errors: List of validation errors
+            warnings: Optional list of validation warnings
+            summary: Optional summary message
+
+        Returns:
+            ValidationResult indicating failure
+        """
+        if not errors:
+            raise ValueError('Failure result must contain at least one error')
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)